topics/transcriptomics/tutorials/rna-seq-reads-to-counts/tutorial.md

```diff
@@ -201,6 +218,15 @@ Sequence quality control is therefore an essential first step in your analysis.
 > 2. Inspect the `Webpage` output of **FastQC** {% icon tool %} for the `MCL1-DL` sample by clicking on the {% icon galaxy-eye %} (eye) icon
 {: .hands_on}
 
+
+> ### {% icon tip %} Tip: Using collections with a tool
```

Comment:
we have a snippet for this now that you can add in hands-on boxes where relevant {% include snippets/select_collection.md %}